extensions/podman/packages/extension/package.json

```diff
@@ -100,14 +104,24 @@
           "maximum": "HOST_TOTAL_DISKSIZE",
           "scope": "ContainerConnection",
           "step": 500000000,
-          "description": "Disk size"
+          "description": "Disk size",
+          "when": "isMac"
         },
         "podman.machine.diskSizeUsage": {
           "type": "number",
           "format": "diskSizeUsage",
           "scope": "ContainerConnection",
           "description": "Disk size usage",
-          "readonly": true
+          "readonly": true,
+          "when": "isMac"
```

Comment:
suggestion: For diskusage, memory, and CPU, might be better to have this in a dedicated PR